### PR TITLE
Tasks Refactor

### PR DESCRIPTION
**Pull request by joaopapereira**
This PR will refactor the Tasks to make them easier to use, and to use the Pagination Framework

### CODE REVIEW

File: app/controllers/company_people_controller.rb [hunk 1]
```diff
@@ -60,9 +60,10 @@ def destroy
   end
 
   def home
-    @task_type_t1 = 'mine-open'
-    @render_modal = true
-    @tasks_t1 = display_tasks @task_type_t1
+    #@task_type_t1 = 'mine-open'
+    #@render_modal = true
+    #@tasks_t1 = display_tasks @task_type_t1
```
Comment (patmbolger) [inline app/controllers/company_people_controller.rb]: Should these 3 lines be removed rather than commented out?

**Comment by patmbolger** [comment]
Reviewed PR with one comment.

**Comment by joaopapereira** [comment]
Now that the test are passing I believe I can merge this request

**Comment by patmbolger** [comment]
PR reviewed ... looks good.

**Pull request merged**